clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -225,7 +225,8 @@ check_sane_control_flow()
             gen_instr(1, 20),
```

Comment:
Please follow the commit description conventions:

+ `i#NNN: ` title prefix
+ `Issue: NNNN` line at the end of the body